core/lib/presentation/utils/responsive_utils.dart

```diff
@@ -51,4 +53,8 @@ class ResponsiveUtils {
       isMobile(context) ? _loginTextFieldWidthSmallScreen : _loginTextFieldWidthLargeScreen;
 
   double getWidthLoginButton() => _loginButtonWidth;
+
+  bool isHeightShortest(BuildContext context) {
```

Comment:
What is this used for?

Reply:
use for padding the top in case the screen have smaller than 600